jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt

```diff
@@ -67,35 +64,23 @@ object JavaCodeWhispererFileCrawler : CodeWhispererFileCrawler() {
         return result
     }
 
-    override fun listFilesWithinSamePackage(targetFile: PsiFile): List<VirtualFile> = runReadAction {
-        targetFile.containingDirectory?.files?.mapNotNull {
-            // exclude target file
-            if (it != targetFile) {
-                it.virtualFile
-            } else {
-                null
-            }
-        }.orEmpty()
-    }
```

Comment:
moved to its parent class `CodeWhispererFileCralwer` since this method is suppoed to be language-agnostic https://github.com/aws/aws-toolkit-jetbrains/pull/3774/files#diff-faff4822410665507967aaa15bfe64abab03286e73a1bdc9c9d5046408de467bR101